demo/pages/elements/form/FormDemo.js

```diff
@@ -117,6 +117,7 @@ export class FormDemoComponent {
         // Dynamic
         this.dynamic = formUtils.toControls(MockMeta, '$ USD', {}, 'TOKEN');
         formUtils.setInitialValues(this.dynamic, { firstName: 'Initial F Name' });
+        formUtils.setInitialValues(this.dynamic, { number: 12 });
```

Comment:
`formUtils.setInitialValues(this.dynamic, { firstName: 'Initial F Name', number: 12 });`